opensrp-giz-malawi/src/main/java/org/smartregister/giz/activity/FragmentBaseActivity.java

```diff
@@ -77,13 +76,13 @@ public void sendEmail() {
                         .fromView(getPDFRootView())
                         .setDefaultPageSize(PdfGeneratorHelper.PageSize.WRAP_CONTENT)
                         .setFileName(emailAttachmentName)
-                        .setFolderName("Reports")
+                        .setFolderName(GizConstants.EmailParameterHelper.REPORTS)
                         .openPDFafterGeneration(false)
                         .build(new PdfGeneratorListener() {
                             @Override
                             public void onFailure(PdfGeneratorHelper.FailureResponse failureResponse) {
                                 super.onFailure(failureResponse);
-                                failureResponse.getThrowable().printStackTrace();
+                                Timber.e("Pdf Could not be Generated");
```

Comment:
Could this be `Timber.e(failureResponse.getThrowable(), "Pdf Could not be Generated");`

Reply:
DONE